CMakeLists.txt

```diff
@@ -139,6 +139,8 @@ string(REGEX MATCH "Clang" CMAKE_COMPILER_IS_CLANG "${CMAKE_C_COMPILER_ID}")
 
 include(CheckCCompilerFlag)
 
+set(CMAKE_C_STANDARD 99)
+
```

Comment:
Hang on a second, this doesn't have `set(CMAKE_C_EXTENSIONS OFF)`. I think extensions are required to pass the CI (apparently we use `time_t` in testing) but I think we should try to address this another way seeing as we'd like to not require any extensions.